setup.py

```diff
@@ -0,0 +1,17 @@
+"setup.py"
+
+from distutils.core import setup
+from distutils.extension import Extension
+from Cython.Build import cythonize
+
+extensions = [
+    Extension("soletta", ["bindings/python/mainloop.pyx"],
+        include_dirs = ["./build/soletta_sysroot/usr/include/soletta"],
```

Comment:
this is not right, you can't assume prefix = `/usr`, for instance I never install soletta to my host system, instead I configure it with `/tmp/test-soletta` and define the `$PATH`, `$LD_LIBRARY_PATH` and `$PYTHONPATH` to make it usable. :-)

I'm not sure we even need to provide this `setup.py` in the top directory. If you want to allow "building from easy_install" or the likes, we need to do a different approach with it calling Soletta's build (kconfig/kbuild).